edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter/handlers/PolicyUpdateHandler.cs

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter
+{
+    using System;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Hub.Core;
+    using Microsoft.Azure.Devices.Edge.Hub.Core.Config;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Responsible for listening for EdgeHub config updates (twin updates)
+    /// and pushing new authorization policy definition from EdgeHub config to the Mqtt Broker.
+    /// </summary>
+    public class PolicyUpdateHandler : IMessageProducer
+    {
+        // !Important: please keep in sync with mqtt-edgehub::command::POLICY_UPDATE_TOPIC
```

Comment:
we should have a test for this to make sure they are in sync

Reply:
I believe there would be not much benefit of such test. I think comment is enough to notify future devs in case they want to change this constant.